frame/bounties/src/benchmarking.rs

```diff
@@ -218,6 +330,130 @@ benchmarks! {
 		ensure!(missed_any == false, "Missed some");
 		assert_last_event::<T>(RawEvent::BountyBecameActive(b - 1).into())
 	}
+
+	add_subbounty {
+		let d in 0 .. MAX_BYTES;
+		setup_pot_account::<T>();
+
+		let bm_setup = create_subbounty_bounty::<T>(0, d)?;
+
+	}: _(
+		RawOrigin::Signed(bm_setup.curator),
+		bm_setup.bounty_id,
+		bm_setup.subbounty_value,
+		bm_setup.reason
+	)
```

Comment:
ideally all benchmark could contains verify just so they don't break unexpectedly. But I'm fine as it is.

Reply:
:+1: